src/builder.ts

```diff
@@ -8,28 +8,29 @@ import { PackageJson } from './interfaces';
 export class Builder {
 
   public static create(): Builder {
-    return new Builder(loadPackageJson, promisify(rimraf), promisify(exec));
+    return new Builder(loadPackageJson, promisify(rimraf), promisify(exec), console);
   }
 
   constructor (
     private loadPackageJson: () => PackageJson,
     private rimraf: (path: string) => Promise<void>,
-    private exec: (command: string) => Promise<{ stdout: string, stderr: string }>
+    private exec: (command: string) => Promise<{ stdout: string, stderr: string }>,
+    private console: { log(string: string): void, error(string: string): void }
```

Comment:
Since console is global available, I would think you can access it in the test already and `spyOn` it